failure in openshift-ansible conformance networking router

#### Description

Apparent test flake; at least, the failures don't look related to the PR.

##### Observed Results

https://ci.openshift.redhat.com/jenkins/job/merge_pull_request_openshift_ansible/182/

aos-ci-test passed, merge test failed with several indicators:

```
[Conformance][networking][router] openshift router metrics
/go/src/github.com/openshift/origin/test/extended/router/metrics.go:171
  The HAProxy router
  /go/src/github.com/openshift/origin/test/extended/router/metrics.go:170
    should expose prometheus metrics for a route [It]
    /go/src/github.com/openshift/origin/test/extended/router/metrics.go:152

    Expected
        <[]float64 | len:1, cap:1>: [6]
    to consist of
        <[]interface {} | len:1, cap:1>: [
            {Comparator: ">=", CompareTo: [10]},
        ]

    /go/src/github.com/openshift/origin/test/extended/router/metrics.go:133
```

```
Summarizing 2 Failures:

[Fail] [Conformance][networking][router] openshift router metrics The HAProxy router [It] should expose prometheus metrics for a route 
/go/src/github.com/openshift/origin/test/extended/router/metrics.go:133

[Fail] [k8s.io] Kubectl client [k8s.io] Kubectl run rc [AfterEach] should create an rc from an image [Conformance] 
/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/test/e2e/framework/util.go:2066

Ran 190 of 726 Specs in 1346.776 seconds
FAIL! -- 188 Passed | 2 Failed | 0 Pending | 536 Skipped 
```

```
Test Suite Passed
[INFO] Dumping container logs to /tmp/openshift/conformance/logs/containers
[INFO] Exiting
[ERROR] PID 26175: test/extended/conformance.sh:51: `[[ -n "${LOG_DIR:-}" && -d "${LOG_DIR-}" ]]` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: test/extended/conformance.sh:51: `[[ -n "${LOG_DIR:-}" && -d "${LOG_DIR-}" ]]`
[INFO]   Exiting with code 1.
make: *** [test-extended] Error 1
++ export status=FAILURE
++ status=FAILURE
+ set +o xtrace
########## FINISHED STAGE: FAILURE: RUN EXTENDED TESTS ##########```
```

--- a/test/extended/router/metrics.go
+++ b/test/extended/router/metrics.go
@@ -94,7 +94,7 @@ var _ = g.Describe("[Conformance][networking][router] openshift router metrics",
 				o.Expect(err).NotTo(o.HaveOccurred())
 				//e2e.Logf("Metrics:\n%s", results)
 				if len(findGaugesWithLabels(metrics["haproxy_server_up"], serverLabels)) == 2 {
-					if findGaugesWithLabels(metrics["haproxy_server_connections_total"], serverLabels)[0] > 0 {
+					if findGaugesWithLabels(metrics["haproxy_backend_connections_total"], routeLabels)[0] >= float64(times) {
 						break
 					}
 					// send a burst of traffic to the router

